crates/rslint_cli/src/panic_hook.rs

```diff
@@ -1,23 +1,36 @@
 //! The custom panic hook used by the linter to issue a more descriptive explanation.
 
-use crate::{lint_err, lint_note};
+use codespan_reporting::term::termcolor::{self, ColorChoice};
+use std::io::Write;
 use std::panic::PanicInfo;
 
 pub fn panic_hook(info: &PanicInfo) {
-    lint_err!("The linter panicked unexpectedly. this is a bug.");
-    eprintln!("We would appreciate a bug report: https://github.com/RDambrosio016/RSLint/issues/new?labels=ILE%2C+bug&template=internal-linter-error.md\n");
-    lint_note!("Please include the following info: ");
+    let stderr = termcolor::StandardStream::stderr(ColorChoice::Always);
```

Comment:
```suggestion
   
```
No point in acquiring a termcolor lock

Reply:
[`StandardStreamLock`](https://docs.rs/termcolor/1.1.0/termcolor/struct.StandardStreamLock.html) relies on the lifetime of the `StandardStream` that it comes from. 
We have to hold `stderr` here to keep the lock alive for the whole function.

Reply:
Yeah but we don't use any termcolor items here, a normal std lock works

Reply:
Won't the text be just normal if we use a `io::stderr()`?

Reply:
It will be normal either ways because we don't use any colors